torchaudio/datasets/cmudict.py

```diff
@@ -169,4 +169,6 @@ def __len__(self) -> int:
 
     @property
     def symbols(self) -> List[str]:
+        """list[str]: A list of phonemes symbols.
```

Comment:
nit: I think it's helpful to add example symbols.
```suggestion
        """list[str]: A list of phonemes symbols. such as `AA`, `AE`, `AH`.
```